src/routes/coupon.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: elvis
+ * Date: 2018/11/11
+ * Time: 18:12
+ */
+
+use \Psr\Http\Message\ServerRequestInterface as Request;
+use \Psr\Http\Message\ResponseInterface as Response;
+
+// Get Coupons By User
+$app->get('/api/coupons/customer/{username}', function (Request $request, Response $response, array $args) {
+    $username = $args['username'];
+    $sql = "SELECT C.id, C.username, C.type_id, C.hotel_id, C.expire_date, CT.value, CT.discount_type 
+            FROM coupon C, coupon_type CT 
+            WHERE username = :username AND C.type_id = CT.id";
```

Comment:
I think you can add `GROUP BY C.hotel_id`, this makes the front-end side easier, and also achieve the project requiremtn